docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -97,93 +161,129 @@ Windows::
 ----
 for /l %x in (1, 1, 1000) do clc -c source.yaml map --name my-map set key-%x value-%x --quiet
 ----
+--
 ====
 
-NOTE: You can also use xref:getting-started:get-started-binary.adoc#step-3-write-data-to-memory[alternative ways] to create a map and put data.
+==== Update the Configuration
 
-=== Step 3. Start the Target Cluster
+You must update the following configuration:
 
-The target cluster must be of version Hazelcast 5.3.2 or newer.
+* The cluster information
+* The data structure information
 
-Run the following Docker command to start a one-member target cluster.
+To update the cluster information, complete the following steps:
 
-[source,shell]
-----
-docker run -e HZ_NETWORK_PORT_PORT=5901 -p 127.0.0.1:5901:5901 
-           -e HZ_LICENSEKEY="<license>" <1>
-           -e HZ_CLUSTERNAME=target hazelcast/hazelcast-enterprise:{full-version}-slim <2>
-----
-<1> If you don't have a Hazelcast Enterprise license key, you can either use a Hazelcast Viridian https://docs.hazelcast.com/cloud/free-trial[trial cluster] or request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license].
-<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/source/hazelcast.yaml_ file in your favorite editor
++
+NOTE: The _hazelcast.yaml_ file is a Hazelcast client configuration file, which can include any supported configuration.
++
+. Update the `cluster-name` field to match the name of your source cluster
+. Update the `cluster-members` field to match the addresses of the cluster members
```

Comment:
The same needs to be done for target, and if the target is Viridian it is more involved - there are keys the user needs to put into the target folder. I will provide a sample.

Reply:
Sample content for hazelcast-client.yaml:

```
hazelcast-client:

  cluster-name: xyz
  network:
    hazelcast-cloud:
      enabled: true
      discovery-token: tokentoken
    ssl:
      enabled: true
      properties:
        keyStore: client.keystore
        keyStorePassword: abc123
        trustStore: client.truststore
        trustStorePassword: abc123
```

The values for `cluster-name`, `discovery-token` and passwords can be obtained from the advanced setup in viridian documented here https://docs.hazelcast.com/cloud/connect-to-cluster#advanced